CI/E2E/docker-compose-local.yaml

```diff
@@ -16,12 +16,35 @@ services:
         "node",
         "-r",
         "dotenv/config",
-        ".",
+        "."
       ]
     volumes:
       - ".env.scichat-loopback:/home/node/app/.env"
     depends_on:
       - mongodb
+
+  es01:
+    depends_on:
+      - mongodb
+    image: docker.elastic.co/elasticsearch/elasticsearch:${STACK_VERSION}
```

Comment:
would this also work with one of the free(er) clones, e.g. opensearch?

Reply:
WE should open a new issue to investigate